xla/stream_executor/sycl/sycl_status.h

```diff
@@ -0,0 +1,51 @@
+/* Copyright (c) 2025 The OpenXLA Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef XLA_STREAM_EXECUTOR_SYCL_SYCL_STATUS_H_
+#define XLA_STREAM_EXECUTOR_SYCL_SYCL_STATUS_H_
+
+#include <string>
+#include <sycl/sycl.hpp>
```

Comment:
I see this includes sycl-specific headers. How do we get this to build on our CI? Do you already have some Bazel build rules that make Bazel build SYCL, or download prebuilt libraries and headers?

(I'm reluctant to accept code that we can't build because it makes refactoring so much harder.)

Reply:
@beckerhe The change to add clang build support as well as hermetic build is being worked on and should be ready soon. We can put this on hold until then.

Reply:
@kanvi-nervana, Could you please mark this PR as draft, until all prerequisites are ready and submitted? Once this PR is ready to be reviewed again, please remove the draft status and ping us here.

Reply:
@dimitar-asenov I have updated the PR since the pre-requisites have been merged. The PR is ready for review. Thanks!